ps2.h

```diff
@@ -2,32 +2,31 @@
 // Copyright (c) 2019 Michael Steil
 // All rights reserved. License: 2-clause BSD
 
-#ifndef _PS2_H_
-#define _PS2_H_
+#ifndef PS2_H
+#define PS2_H
 
 #include <stdint.h>
 
-#define PS2_DATA_MASK 1
-#define PS2_CLK_MASK 2
+#define PS2_DATA_MASK (0x01)
+#define PS2_CLK_MASK (0x02)
+#define PS2_VIA_MASK (0x03)
 
 typedef struct {
-	int clk_out;
-	int data_out;
-	int clk_in;
-	int data_in;
+	uint8_t out;
+	uint8_t in;
 } ps2_port_t;
 
 extern ps2_port_t ps2_port[2];
 
-bool ps2_buffer_can_fit(int i, int n);
 void ps2_buffer_add(int i, uint8_t byte);
-void ps2_step(int i);
+void ps2_step(int i, int clocks);
+void ps2_autostep(int i);
 
-// fake mouse
-void mouse_button_down(int num);
-void mouse_button_up(int num);
-void mouse_move(int x, int y);
-void mouse_send_state();
+    // fake mouse
+void    mouse_button_down(int num);
```

Comment:
please no indentation to line up function names

Reply:
Will try to keep this in mind in the future. I blame clang-format.